From 990843643ca68c40b5e9371212c775054eb19b95 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Thu, 20 Jun 2019 20:09:50 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Set boot-on property to vdd_arm
 regulator

This is used to keep the stock voltage specified by bootloader.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index 2e40b051546f..e9a05dccb734 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -842,6 +842,7 @@
 			regulator-min-microvolt = <770000>;
 			regulator-max-microvolt = <1100000>;
 			regulator-always-on;
+			regulator-boot-on;
 		};
 	};
 
-- 
2.22.0

